cmd/katapult/data_centers_test.go

```diff
@@ -0,0 +1,136 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	"github.com/krystal/go-katapult"
+	"github.com/krystal/go-katapult/core"
+	"github.com/krystal/katapult-cli/cmd/katapult/mocks"
+
+	"github.com/stretchr/testify/assert"
+)
+
+var dcs = []*core.DataCenter{
+	{
+		ID:        "POG1",
+		Name:      "hello",
+		Permalink: "Hello World!",
+		Country: &core.Country{
+			ID:   "POG",
+			Name: "Pogland",
+		},
+	},
+	{
+		ID:        "GB1",
+		Name:      "hello",
+		Permalink: "Hello World!",
+		Country: &core.Country{
+			ID:   "UK",
+			Name: "United Kingdom",
+		},
+	},
+}
+
+const stdoutDcList = ` - hello (Hello World!) [POG1] / Pogland
+ - hello (Hello World!) [GB1] / United Kingdom
+`
+
+func TestDataCenters_List(t *testing.T) {
+	mock := singleResponse(t, "/core/v1/data_centers", "data_centers", dcs)
+	cmd := dataCentersCmd(mock)
+	stdout := &bytes.Buffer{}
+	cmd.SetOut(stdout)
+	cmd.SetArgs([]string{"list"})
+	if err := cmd.Execute(); err != nil {
+		t.Fatal(err)
+	}
+	assert.Equal(t, stdoutDcList, stdout.String())
+}
+
+func TestDataCenters_Get(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	mock := mocks.NewMockRequestMaker(ctrl)
+	matcher := mocks.KatapultRequestMatcher{
+		Path: "/core/v1/data_centers/_",
+		ExpectedParams: []mocks.URLParamMatcher{
+			mocks.URLParamContains("data_center[permalink]"),
+		},
+	}
+	tests := []struct {
+		name string
+
+		args   []string
+		dc     string
+		wants  string
+		stderr string
+		err    string
+	}{
+		{
+			name: "display POG1",
+			args: []string{"get", "POG1"},
+			wants: `hello (Hello World!) [POG1] / Pogland
+`,
```

Comment:
This is a personal thing, so feel to disagree :)... I just dislike how raw strings mess with indentation, so for a single thing like this, I prefer a double quoted string with `\n` at the end to ensure the string ends with a line feed.

```suggestion
			wants: "hello (Hello World!) [POG1] / Pogland\n",
```